opentech/apply/review/models.py

```diff
@@ -1,17 +1,59 @@
 from django.conf import settings
+from django.contrib.postgres.fields import JSONField
 from django.db import models
 
-from django.contrib.postgres.fields import JSONField
+
+from opentech.apply.users.models import User
+
+NO = 0
+MAYBE = 1
+YES = 2
 
 RECOMMENDATION_CHOICES = (
-    (0, 'No'),
-    (1, 'Yes'),
-    (2, 'Maybe')
```

Comment:
Ugh, this becomes yet another value map to migrate. In the old site "maybe" is 2

Reply:
True, but it feels more reasonably to do it this way. If we are migrating it is there another approach you would rather do that might work better?

Reply:
had a thought and I can't find a better way, so let's leave it as is.